browser/scripts/ui.js

```diff
@@ -621,6 +621,87 @@ VizorUI.prototype.updateProgressBar = function(percent) {
 
 /***** HELPER METHODS *****/
 
+VizorUI.openEditorHelp = function() {
+	var keyData = _.extend({}, uiKeys);
+	var mod_shift = uiKeys.mod_shift,
+	 	mod_ctrl = uiKeys.mod_ctrl,
+	 	mod_alt = uiKeys.mod_alt;
+
+	var htmlFromKeyCode = function(modifiedKeyCode) {
+		var isOSX = /mac os x/.test(navigator.userAgent.toLowerCase());
+		var key='', html = []
+		var add = function(key, className) {
+			var tag
+			if (typeof className !== 'undefined')
+				tag = '<kbd class="' + className + '">'
+			else
+				tag = '<kbd>';
+			var end = '</kbd>'
+			html.push(tag + key.toLowerCase() + end)
+		}
+
+		if (Object.prototype.toString.call(modifiedKeyCode) === '[object String]' ) { // isString()
+			key = modifiedKeyCode;
+			add(key.toLowerCase());
+		} else {
+			var key_meta = (isOSX) ? 'cmd' : 'ctrl';
+
+			if (modifiedKeyCode >= mod_alt) {
+				add('alt', 'modifier key_alt');
+				modifiedKeyCode -= mod_alt;
+			}
+			if (modifiedKeyCode >= mod_ctrl) {
+				add(key_meta, 'modifier key_ctrl key_cmd');
+				modifiedKeyCode -= mod_ctrl;
+			}
+			if (modifiedKeyCode >= mod_shift) {
+				add('shift', 'modifier key_shift');
+				modifiedKeyCode -= mod_shift;
+			}
+
+			var keyCodeMap = {	// keys without shift
+				186: 59,
```

Comment:
Good point. I'd like to stop processing keycodes in this primitive way altogether. Nothing I've found so far has been reliable.